stock_scanner_inventory/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# © 2011-2015 Sylvain Garancher <sylvain.garancher@syleam.fr>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Stock Scanner Inventory',
+    'version': '11.0.1.0.0',
+    'category': 'Generic Modules/Inventory Control',
+    'website': 'https://odoo-community.org/',
```

Comment:
please change website to: https://github.com/OCA/stock-logistics-barcode

Reply:
file updated.